package.json

```diff
@@ -22,6 +22,7 @@
     "postinstall": "polkadot-dev-yarn-only",
     "test": "jest --coverage --testPathIgnorePatterns e2e",
     "test:all": "jest",
+    "test:one": "jest",
```

Comment:
`jest test singlefile.spec.ts` does not work anymore, well, this is probably better anyway since it doesn't add the `--coverage` stuff, which, when running single, you don't care about.

Reply:
There's some trick by using something like
```
"test": "jest --coverage --testPathIgnorePatterns e2e --"
```

then single files still work